config/routes.rb

```diff
@@ -392,6 +392,7 @@
 
     get '/account/verify' => 'idv/gpo_verify#index', as: :idv_gpo_verify
     post '/account/verify' => 'idv/gpo_verify#create'
+    get '/account/cancel_verify' => 'idv/gpo_confirm_cancel#index', as: :idv_gpo_confirm_cancel
```

Comment:
I saw there was a discussion of the path to use, but I don't remember how it turned out. This one seems ambiguous. Are we cancelling verification, or verifying cancellation? Maybe `/account/confirm_cancel`.

Reply:
Related, I'd be in favor of aligning the URL path to the controller name as much as possible, to make it easier to track as a developer, and since we shouldn't need to have two different names to describe what the controller is for.

i.e. either `/account/gpo_confirm_cancel` (update the route) or `/account/confirm_cancel` + `idv/confirm_cancel` (update the controller)

Reply:
I vaguely recall some discussion of avoiding the use of "gpo" in anything that is public facing, but I also think we should just do it in this case. We need to keep the controller name specific, because there is already a separate cancellations controller for other parts of the idv process

Reply:
per the discussion with @jmhooper and @benjaminchait, we had landed on `account/verify/start_over`

Reply:
@kbighorse We should discuss, because the gpo verify controller uses `/account/*` rather than `/verify/*`